Formula/questdb.rb

```diff
@@ -0,0 +1,64 @@
+class Questdb < Formula
+  desc "Time Series Database"
+  homepage "https://www.questdb.org"
+  url "https://www.questdb.org/download/questdb-1.0.0-SNAPSHOT-20161013-1304-bin.tar.gz"
```

Comment:
Can we let this pull thru with snapshot version? I realise we need a release, but i'm not quite ready with all of the testing.